src/view-data/responseType.ts

```diff
@@ -1,9 +1,20 @@
 import { convertType } from "../typescript";
 import { HttpOperation, Swagger, SwaggerType } from "../swagger/Swagger";
+import { values, uniq } from "lodash/fp";
 
-const defaultSuccessfulResponseType = "void";
+const defaultResponseType = "void";
+
+export const getResponseTypes = (op: HttpOperation, swagger: Swagger): string =>
+  uniq(
```

Comment:
Can we make this slightly more readable with a few comments and helper consts?

Reply:
Added JSDocs and helper functions